projects/plugins/jetpack/extensions/blocks/dialogue/attributes.js

```diff
@@ -1,15 +1,21 @@
 export default {
-	participantSlug: {
-		type: 'string',
+	participant: {
+		type: 'object',
```

Comment:
I'm not sure whether the dialogue attribute should be stored in an object. I suggest using strings instead. Also, I wouldn't pay too much attention to the style's attributes. Provably, they are going to be discarded.

Reply:
Thanks @retrofox! I cherry-picked [your commit](https://github.com/Automattic/jetpack/pull/18598/commits/c126cff69d03723a1d498808457cf5c4d3f87c29) from #18646, as you suggested, to update this.